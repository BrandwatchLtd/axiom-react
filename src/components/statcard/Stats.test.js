import React from 'react';
import renderer from 'react-test-renderer';
import { Stats } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <Stats { ...props }>Lorem</Stats>
  );
}

describe('Stats', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
