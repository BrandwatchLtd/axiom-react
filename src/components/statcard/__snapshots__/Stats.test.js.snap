// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats renders with defaultProps 1`] = `
<ul
  className="ax-stat-cards__stats"
>
  Lorem
</ul>
`;
